relay-event-normalization/src/event.rs

```diff
@@ -373,14 +373,20 @@ pub fn normalize_user_geoinfo(geoip_lookup: &GeoIpLookup, user: &mut User) {
     }
 }
 
-fn normalize_user(user: &mut Annotated<User>) {
-    let Annotated(Some(user), _) = user else {
+fn normalize_user(event: &mut Event) {
+    let Annotated(Some(user), _) = &mut event.user else {
         return;
     };
+
     if !user.other.is_empty() {
         let data = user.data.value_mut().get_or_insert_with(Object::new);
         data.extend(std::mem::take(&mut user.other));
     }
+
+    // We set the `sentry_user` field in the `Event` payload in order to have it ready for the extraction
+    // pipeline.
+    let eventuser_tag = get_eventuser_tag(user);
```

Comment:
Please also replace other uses of `get_eventuser_tag` with direct field access. We can even make the function private then.

Reply:
Are we 100% sure that all the code paths where it's used, are happening afterwards light normalization? I checked the code and it seems like so.